assets/sass/protocol/includes/mixins/_gradients.scss

```diff
@@ -1,54 +1,54 @@
+/* stylelint-disable declaration-block-no-duplicate-properties */
```

Comment:
Removing the `-webkit` vendor prefixes for `background-image` in this file has created duplicate properties, but it looks like only the second occurrence (the standard-based rule) is now valid.

If we fix the duplicates here, can we remove this comment?